src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/BreakEndVariantType.java

```diff
@@ -40,21 +61,51 @@ public final String toString() {
         this.isTheUpstreamMate = isTheUpstreamMate;
     }
 
-    private static Allele constructAltAllele(final byte[] bases, final boolean bracketPointsLeft, final SimpleInterval novelAdjRefLoc,
-                                             final boolean basesFirst) {
-        final String s = bracketPointsLeft ? "]" + novelAdjRefLoc.getContig() + ":" + novelAdjRefLoc.getStart() + "]"
-                                           : "[" + novelAdjRefLoc.getContig() + ":" + novelAdjRefLoc.getStart() + "[";
-        return Allele.create( basesFirst ? new String(bases) + s  : s + new String(bases) );
-    }
-
-    static abstract class StrandSwitchBND extends BreakEndVariantType {
+    //==================================================================================================================
+    public static final class InvSuspectBND extends BreakEndVariantType {
```

Comment:
Why go from `StrandSwitchBND` to `InvSuspectBND`? The old name seemed pretty clear to me.

Reply:
This was to distinguish it from the "translocation BND" type below, because here I'm making the distinction upstream of this step:
* if a contig has two same-chr mappings with strand switch, I'm going to flag them later as inversion suspects (INV33 or INV55), because inversion is kind of special and to be backward compatible (when this experimental tool is hooked up into our main pipeline, inversion calls are not made directly by alignments, but needs other evidence for final call); 
* but for contigs with mappings to different chromosomes, I don't make a distinction between with or without strand switch&mdash;they are going to be output as simple BND formatted records.

So the old name `StrandSwitchBND` was not distinguishing between the inversion suspects (same chr event), and the different-chr events (called translocation here).

Reply:
In retrospect, I think there's a valid concern here: that the inversion breakpoint suspects may turn out to be insertions, and these should be merged with the translocation records, so the "inversion suspect" label should be revoked from such records.
What about dealing with this merge later, as this an issue that is IMO better treated in a different PR?

Reply:
But comments are added to this class for clarifying the distinction.

Reply:
What if you just called them `IntraChromStrandSwitchBND`? That way the logic isn't tied into the name -- IMO the name should just describe what the things _are_, not what you are going to do with them.